<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_unsupervised</class>
 <widget class="QDialog" name="Dlg_unsupervised">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>非监督分类</string>
  </property>
  <widget class="QLabel" name="label_g">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>G波段</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_b">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>B波段</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>580</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_r">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_r">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>R波段</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_b">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_g">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_b_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入迭代次数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_b_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入精确度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_b_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入聚类的最终数目</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
